lib/event/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "vector-event"
+version = "0.1.0"
+authors = ["Vector Contributors <vector@timber.io>"]
+edition = "2018"
+
+[features]
+lua = ["rlua"]
+
+[build-dependencies]
+prost-build = "0.6.1"
+
+[dependencies]
+bytes05 = { package = "bytes", version = "0.5" }
```

Comment:
Perhaps for the common crates we intend to publish we should lock versions, but how will we sync versions for updates between crates of the workspace?

I think we could maybe learn some lessons from how Habitat (https://github.com/habitat-sh/habitat) uses workspace dependencies. This was an issue on the TiKV repo. :(